server/v2/middlewares/decodeToken.js

```diff
@@ -0,0 +1,22 @@
+import jwt from 'jsonwebtoken';
+
+export default function checkAdmin (req, res, next) {
+  let token = req.headers['x-access-token'] || req.headers.authorization;
+
+  if(token){
+    jwt.verify(token, 'secret', (err, decoded) => {
+      if(err) {
+        res.statut(401).send({
```

Comment:
`res.statut`? Ensure you test all these things with Postman before you commit and push